Software/Python/easygopigo3.py

```diff
@@ -367,45 +369,52 @@ def spin_left(self):
         Default ``speed`` is set to **300** - see :py:meth:`~easygopigo3.EasyGoPiGo3.__init__`.
 
         .. important::
-             | This differs from the left() method as both wheels will be rotating but in different directions.
-             | This causes the robot to spin in place, as if doing a pirouette
+             This is a particularity of :py:meth:`~easygopigo3.EasyGoPiGo3.steer` method, thus you can achieve the same by calling ``steer(-100, 100)``.
 
         """
         self.set_motor_dps(self.MOTOR_LEFT, self.get_speed() * -1)
         self.set_motor_dps(self.MOTOR_RIGHT, self.get_speed() )
 
     def steer(self, left_percent, right_percent):
         """
-        Control each motor in order to get a variety of turning movements. 
+        Control each motor in order to get a variety of turning movements.
+
         Each motor is assigned a percentage of the current speed value. 
-        While there is no limit on the values of `left_percent` and `right_percent`, they are expected to be between -100 and 100
+        While there is no limit on the values of ``left_percent`` and ``right_percent`` parameters,
+        they are expected to be between **-100** and **100**.
 
-        @param int left_percent: percentage of current speed value that gets applied to left motor
-        @param int right_percent: percentage of current speed value that gets applied to right motor
+        :param int left_percent: Percentage of current speed value that gets applied to left motor. The range is between **-100** (for backward rotation) and **100** (for forward rotation).
+        :param int right_percent: Percentage of current speed value that gets applied to right motor. The range is between **-100** (for backward rotation) and **100** (for forward rotation).
 
         For setting the motor speed, use :py:meth:`~easygopigo3.EasyGoPiGo3.set_speed`.
         Default ``speed`` is set to **300** - see :py:meth:`~easygopigo3.EasyGoPiGo3.__init__`.
 
         .. important::
-             | Setting both left_percent and right_percent to 100 will result in the same behavior as the forward() method.
-             | Setting both left_percent and right_percent to 0 will stop the GoPiGo
+             Setting both ``left_percent`` and ``right_percent`` parameters to **100** will result in the same behavior as the :py:meth:`~easygopigo3.EasyGoPiGo3.forward` method.
+             The other behavior for :py:meth:`~easygopigo3.EasyGoPiGo3.backward` method will be experienced if both parameters are set to **-100**.
+             
+             Setting both ``left_percent`` and ``right_percent`` to **0** will stop the GoPiGo from moving.
 
         """
         self.set_motor_dps(self.MOTOR_LEFT, self.get_speed() * left_percent / 100)
         self.set_motor_dps(self.MOTOR_RIGHT, self.get_speed() * right_percent / 100)
 
 
 
-    def orbit(self, degrees, radius_cm=0, blocking=True):
+    def orbit(self, degrees, radius_cm = 0, blocking = True):
```

Comment:
no spaces around the = sign, when handling default values in method definitions.
See https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Reply:
PEP8 is a really great style (standard), but only if it's accepted and formalized within the team. Since I've joined DexterInc, no one told me of using such a standard. Even more than this, a while ago (say more than just a few months), I was once corrected for not having added spaces around equals. And since I've known myself, I always added spaces around equals - never the other way. As a consequence, this code standard was never declared a standard in our team and nor was I notified of doing something wrong when I added them.

Reply:
Nowhere else in this file you will find spaces around an equal sign in the method declaration. At the minimum, you're changing the standard of this particular file.

As for Pep-8, we loosely follow it. The more a file is meant to be seen by a user the closer we follow it. This one here is not meant for the end user, but at least it should be internally consistent.

Reply:
Then why did no one tell me about this? Simple as that. If the standard is accepted within the team and I'm a part of the team, then I must have known of this from ever since I got onboard.